go/oasis-node/cmd/registry/entity/entity.go

```diff
@@ -33,6 +33,7 @@ const (
 	cfgAllowEntitySignedNodes = "entity.debug.allow_entity_signed_nodes"
 	cfgNodeID                 = "entity.node.id"
 	cfgNodeDescriptor         = "entity.node.descriptor"
+	cfgSignedFile             = "entity.signed_tx_file"
```

Comment:
Consistency with staking CLI:
```suggestion
	cfgTxFile             = "entity.transaction.file"
```